app/views/assignments/_row.html.erb

```diff
@@ -1,12 +1,16 @@
 <% route = course_assignment_path(@current_course, assignment.id) %>
 <% assignment_text = "#{h(assignment.short_identifier)}: #{h(assignment.description)}" %>
+<% grace_credits_used = @current_role.grace_credits_used_for(assignment) %>
 <tr>
   <td>
     <%= link_to assignment_text, route %>
   </td>
   <td>
     <%= render partial: 'assignments/assignment_date',
                locals: { assignment: assignment } %>
+    <% if grace_credits_used > 0 %>
+      <span><%= I18n.t('assignments.grace_credits_used', grace_credits_used: grace_credits_used).html_safe%></span>
```

Comment:
It's a good instinct to use `html_safe` here, but there's a more idiomatic Rails way to achieve this. See https://guides.rubyonrails.org/i18n.html#using-safe-html-translations

Reply:
This is the kind of insight and maturity that I hope to develop one day! I initially used I18n.t because I saw it being used within `_assignment_date.html.erb`. When the HTML tags were being escaped, I looked on StackOverflow for an I18n-specific solution! It never came to my mind that I might be overthinking things 😪

Reply:
@hemmatio oh honestly I'm not sure it was "insight" or "maturity", I just happened to know about this because of experience working with Rails, and I suppose reading through the guide. There's a lot of great info in these guides!